soc/hps_lattice_nx.py

```diff
@@ -0,0 +1,174 @@
+##################################################################
+# This is a modified version of litex/soc/cores/ram/lattice_nx.py
+# Modified by CFU Playground Authors to:
+#  * optionally use dual port LRAM primitive
+#  * optionally stripe consecutive addresses across the banks
+##################################################################
+#
+#
+# This file is part of LiteX.
+#
+# Copyright (c) 2020 David Corrigan <davidcorrigan714@gmail.com>
```

Comment:
Copyright should remain as first thing in the file.

Reply:
Fixed.

Reply:
Also added CI tests with this option enabled (failure is masked, but that should get addressed soon, see #365 ).

Also added some ascii art in `hps_lattice_nx.py` showing the memory org.